File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/wishList/EntryRating.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.wishList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryRating {

    @SerializedName("rated")
    @Expose
    private String rated;
    @SerializedName("rating_num")
    @Expose
    private String ratingNum;
    @SerializedName("rating_num_float")
    @Expose
    private Long ratingNumFloat;
    @SerializedName("rating")
    @Expose
    private String rating;

    public String getRated() {
        return rated;
    }

    public void setRated(String rated) {
        this.rated = rated;
    }

    public String getRatingNum() {
        return ratingNum;
    }

    public void setRatingNum(String ratingNum) {
        this.ratingNum = ratingNum;
    }

    public Long getRatingNumFloat() {
        return ratingNumFloat;
    }

    public void setRatingNumFloat(Long ratingNumFloat) {
        this.ratingNumFloat = ratingNumFloat;
    }

    public String getRating() {
        return rating;
    }

    public void setRating(String rating) {
        this.rating = rating;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itemlist/EntryRating.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryRating {

    @SerializedName("rating_num_float")
    @Expose
    var ratingNumFloat: Int = 0
    @SerializedName("rating_num")
    @Expose
    var ratingNum: String? = null
    @SerializedName("rated")
    @Expose
    var rated: String? = null
    @SerializedName("rating")
    @Expose
    var rating: String? = null

}
